### PR TITLE
Add personal Tada68 Keymap

### PR DESCRIPTION
**Pull request by thefezzant**

### CODE REVIEW

**Comment by drashna** [comment]
Could you your keymap folder to use only lowercase characters?  We're trying to keep it consistent for OS's that may have issues with that.

If needed, rename to a temp name, commit and then rename back and commit.

**Comment by thefezzant** [comment]
Still learning git, but hopefully that did it

**Comment by drashna** [comment]
No worries. It's a bit of a steep learning curve! 

As for the `config.h`, you don't need the whole keyboard file.  In fact, this should be all you need: 
```
#ifndef CONFIG_USER_H
#define CONFIG_USER_H

#include QMK_KEYBOARD_CONFIG_H

/* Backlight configuration
 */
#define BACKLIGHT_LEVELS 6

/* Tapping Term variable
 */
#define TAPPING_TERM 200

#endif
```

As these are the only settings that you changed, from the looks of it, that's all you need.  Note the `CONFIG_USER_H` rather than `CONFIG_H`.

File: keyboards/tada68/keymaps/fezzant/config.h [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef CONFIG_H
```
Comment (drashna) [inline keyboards/tada68/keymaps/fezzant/config.h]: This file should be just: 
```c
#ifndef CONFIG_USER_H
#define CONFIG_USER_H

#include QMK_KEYBOARD_CONFIG_H

/* Backlight configuration
 */
#define BACKLIGHT_LEVELS 6

/* Tapping Term variable
 */
#define TAPPING_TERM 200

#endif
```

**Comment by drashna** [comment]
You can still use a `config.h` file, but it should be contain very minimal settings, as listed above.

Otherwise, is this ready?

**Comment by thefezzant** [comment]
Yes, it is ready. `config.h` was not needed since I don't believe it was being compiled in the first place

**Comment by drashna** [comment]
It was, but it wouldn't have been used, because of how it was formatted (the `#ifndef CONFIG_H` part). 

However, is you need to add customization, the above block that I've posted should work fine.

Either way, thanks!

**Pull request merged**